tests/Integration/DashboardLinksTest.php

```diff
@@ -0,0 +1,150 @@
+	 *
+	 * @since 2.6.0
+	 * @since 3.2.0 Moved to `DashboardLinksTest.php`
+	 *
+	 * @covers \Parsely\UI\Row_Actions::cannot_show_parsely_link
+	 * @uses \Parsely\Parsely::api_key_is_set
+	 * @uses \Parsely\Parsely::api_key_is_missing
+	 * @uses \Parsely\Parsely::get_options
+	 * @uses \Parsely\Parsely::post_has_trackable_status
+	 * @uses \Parsely\Parsely::update_metadata_endpoint
+	 * @group ui
+	 */
+	public function test_can_correctly_determine_if_Parsely_link_can_be_shown_when_actions_are_an_array_or_not(): void {
+		$published_post = self::factory()->post->create_and_get();
+		self::set_options( array( 'apikey' => 'somekey' ) );
+
+		self::assertFalse( Dashboard_Links::cannot_show_link( $published_post, self::$parsely ) );
+	}
+
+	/**
+	 * Test if logic for showing Parse.ly row action accounts for post having trackable status.
+	 *
+	 * @since 2.6.0
+	 * @since 3.2.0 Moved to `DashboardLinksTest.php`
+	 *
+	 * @covers \Parsely\UI\Row_Actions::cannot_show_parsely_link
+	 * @uses \Parsely\Parsely::api_key_is_set
+	 * @uses \Parsely\Parsely::api_key_is_missing
+	 * @uses \Parsely\Parsely::get_options
+	 * @uses \Parsely\Parsely::post_has_trackable_status
+	 * @uses \Parsely\Parsely::update_metadata_endpoint
+	 * @group ui
+	 */
+	public function test_can_correctly_determine_if_Parsely_link_can_be_shown_when_post_has_trackable_status_or_not(): void {
+		$draft_post     = self::factory()->post->create_and_get( array( 'post_status' => 'draft' ) );
+		$published_post = self::factory()->post->create_and_get();
+
+		self::set_options( array( 'apikey' => 'somekey' ) );
+
+		// Test if post does not have trackable status - only published posts are tracked by default.
+		self::assertTrue( Dashboard_Links::cannot_show_link( $draft_post, self::$parsely ) );
+		self::assertFalse( Dashboard_Links::cannot_show_link( $published_post, self::$parsely ) );
+	}
+
+	/**
+	 * Test if logic for showing Parse.ly row action accounts for post not having a viewable type.
+	 *
+	 * @since 2.6.0
+	 * @since 3.2.0 Moved to `DashboardLinksTest.php`
+	 *
+	 * @covers \Parsely\UI\Row_Actions::cannot_show_parsely_link
+	 * @uses \Parsely\Parsely::api_key_is_set
+	 * @uses \Parsely\Parsely::api_key_is_missing
+	 * @uses \Parsely\Parsely::get_options
+	 * @uses \Parsely\Parsely::post_has_trackable_status
+	 * @uses \Parsely\Parsely::update_metadata_endpoint
+	 * @group ui
+	 */
+	public function test_can_correctly_determine_if_Parsely_link_can_be_shown_when_post_is_viewable_or_not(): void {
+		$non_publicly_queryable_post = self::factory()->post->create_and_get( array( 'post_type' => 'parsely_tests_pt' ) );
+		$published_post              = self::factory()->post->create_and_get();
+
+		self::set_options( array( 'apikey' => 'somekey' ) );
+
+		// Test if post is not viewable status.
+		self::assertTrue( Dashboard_Links::cannot_show_link( $non_publicly_queryable_post, self::$parsely ) );
+		self::assertFalse( Dashboard_Links::cannot_show_link( $published_post, self::$parsely ) );
```

Comment:
This assertion is testing exactly the same thing as line 97 and 72. i.e. a viewable and trackable (not sure how a post can be trackable if it's not viewable...) post with an array of row actions.